modules/client-configuration/pages/cli-client-proxy.adoc

```diff
@@ -0,0 +1,54 @@
+[[cli-client-proxy]]
+== Register on the Command Line (Salt)
+
+// Might need an 'unsupported' note? LKB 2019-05-01
+// cf. https://bugzilla.suse.com/show_bug.cgi?id=1131398
+// I'd say "no", according to the outcome of
+// https://github.com/SUSE/spacewalk/issues/9333 KE 2019-12-17
+
+
+Instead of the {webui}, you can use the command line to register a Salt client to a proxy.
+This procedure requires that you have installed the Salt package on the Salt client before registration.
+For SLE 12 based clients, you also must have activated the [systemitem]``Advanced Systems Management`` module.
+
+[NOTE]
+====
+Registring on the command line is also possible with traditional clients, but it requires more steps.
+It is not covered here.
+Use the bootstrap script procedure to register traditional clients.
+For more information, see xref:script-client-proxy.adoc[].
+====
+
+
+
+.Procedure: Registering Clients to a Proxy Using the Command Line
+
+. Choose a client configuration file located at:
++
+----
+/etc/salt/minion
+----
++
+or:
++
+----
+/etc/salt/minion.d/NAME.conf
+----
++
+This is sometimes also called minion file.
+. Add the proxy FQDN as the `master` to the client configuration file:
++
+----
+master: PROXY123.EXAMPLE.COM
+----
+
+. Save and restart the [systemitem]``salt-minion`` service:
```

Comment:
The 'save' part of this step belongs to the previous step.

Reply:
good spot :)